client/components/cards/attachments.js

```diff
@@ -20,6 +27,28 @@ Template.attachmentsGalery.events({
   'click .js-remove-cover'() {
     Cards.findOne(this.cardId).unsetCover();
   },
+  'click .js-preview-image'() {
+    Popup.open('previewAttachedImage').apply(this, arguments);
+
+    setTimeout(function(){
+      const img = $('img.preview-large-image');
+      const w = img.width(), h = img.height();
+      if(w > 300) {
+        $('div.pop-over').css({
+          width: (w + 20),
+          position: 'absolute',
+          left: (window.innerWidth - w)/2,
+          top: (window.innerHeight - h)/2,
+        });
+      }
+    }, 50);
```

Comment:
Could you explain the reason for this delay — and is there any chance we could remove it at some point?